src/types/index.ts

```diff
@@ -665,3 +666,12 @@ export interface IDefaultComposableOptions {
 }
 
 export type StatusIconType = typeof ALLOWED_ICON_STATUSES[number];
+
+export type ITransactionElement = ITransaction | IActiveMultisigTx
```

Comment:
It would make more sense if the multisig transaction interface be renamed to `IMultisigTransaction`. In our terminology `TX` is something encapsulated inside the `Transaction`.

Also I suggest to rename `ITransactionElement` to something more precise. Word `Element` suggests that this interface describes sub-group of the `Transaction` which is not true. So maybe `IGeneralTransaction` or `ICommonTransaction`?

Last thing I suggest is to move this and the next interface closer to the `ITransaction` interface to have them as one group.